CHANGELOG.md

```diff
@@ -3,6 +3,19 @@
 All significant changes to this repo will be summarized in this file.
 
 
+## [1.8.18](https://github.com/puppetlabs/puppet-resource_api/tree/1.8.18) (2023-07-17)
+
+[Full Changelog](https://github.com/puppetlabs/puppet-resource_api/compare/1.8.16...1.8.18)
+
+**Merged pull requests:**
+
+- \(PA-5641\) Add release job via PR [\#321](https://github.com/puppetlabs/puppet-resource_api/pull/321) ([tvpartytonight](https://github.com/tvpartytonight))
```

Comment:
hmm, I have removed repetitive PRs that are #322 and #323, but maybe I should have smushed them into this line instead of removing them entirely?